docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -67,9 +67,55 @@ Using the PAT authentication requires only the `password` key. The `username` sh
 
 This secret will be used by the build service to perform both default and Pipeline-as-Code builds.
 
+
+It is also possible to have secrets for per-repository or organization access. To do this, a `appstudio.redhat.com/scm.repository` annotation should be added to the secret. It may either specify the full repository path or the
+partial path with a wildcard. For example, to create a secret for all repositories in the `my-user` organization, it should be created as follows:
```

Comment:
```suggestion
partial path with a wildcard. For example, to create a secret for all repositories in the `my-user` organization, create (or add) the following YAML file:
```

I mean that we need an imperative-based phrase "to create a secret..., do this..."